src/ci/docker/scripts/x86_64-gnu-llvm.sh

```diff
@@ -6,10 +6,15 @@ set -ex
 if [[ -z "${PR_CI_JOB}" ]]; then
     # When running gcc backend tests, we need to install `libgccjit` and to not run llvm codegen
     # tests as it will fail them.
+    # The tests are run a second time with the size optimizations enabled.
     if [[ "${ENABLE_GCC_CODEGEN}" == "1" ]]; then
         ../x.py --stage 1 test --skip src/tools/tidy --skip tests/codegen
+        ../x.py --stage 1 test library/std library/alloc library/core \
+            --rustc-args "--cfg feature=\"optimize_for_size\""
     else
         ../x.py --stage 1 test --skip src/tools/tidy
+        ../x.py --stage 1 test library/std library/alloc library/core \
+            --rustc-args "--cfg feature=\"optimize_for_size\""
```

Comment:
The invocation is the same in the 2 branch so it should just be moved outside of it.

Reply:
Ah, good point, let's just move this outside the `if` block. Otherwise LGTM.

Reply:
Ah yes. Fixed